### ISSUE TITLE
Pass through chat.completions logprobs, top_logprobs, and optional full logits

### ISSUE THREAD

**Issue by electrobrian** [brianreborn/green-roomz#22]
## Why

Standard OpenAI `logprobs` / `top_logprobs` is how a client recovers next-token parameters without a custom dump. OBLITERATUS recovered a 16-token table **exactly** (max abs error 0) when the gateway returned full-vocab `top_logprobs`, and recovered a rank-8 bilinear `embed @ unembed.T` when completions also included raw logits.

llama.cpp already has `n_probs`, completion logprobs, and (on `/completion`) full logits. Green-Roomz `/v1/chat/completions` should not drop them.

## Contract

Honor on `POST /v1/chat/completions`:

- `logprobs: true`
- `top_logprobs: N` (OpenAI cap is 20; if llama.cpp can do more, allow a vendor escape)

Response (OpenAI-shaped):

```json
"choices": [{
  "message": {"role": "assistant", "content": "..."},
  "logprobs": {
    "content": [{
      "token": "Hello",
      "logprob": -0.4,
      "top_logprobs": [{"token": "Hello", "logprob": -0.4}, {"token": "Hi", "logprob": -1.2}]
    }]
  }
}]
```

Optional vendor extension when the client sends `"obliteratus": {"logits": true}` (or `logprobs: "full"`):

```json
"obliteratus": {"logits": [/* vocab-sized float row for the generated step */]}
```

Do not put multi-million-dim logits on the default path. Gate full logits behind the flag. Streamed chunks should carry per-token logprobs when `stream: true`.

## Acceptance

- With `logprobs: true`, a completion includes `choices[0].logprobs.content`.
- `top_logprobs` length matches the request (or llama.cpp max, documented).
- Without the flag, payload size stays as today (no surprise vocab dumps).
- Node test with a fake upstream that returns `n_probs` mapped into the OpenAI shape.

Refs: OpenAI chat logprobs; llama.cpp `n_probs`; OBLITERATUS `chat_completions(..., logprobs=True, top_logprobs=...)`.

**Comment by electrobrian** [comment]
Implemented in https://github.com/brianreborn/green-roomz/pull/24 (release/1.0.0). Live e2e covered this; fork tag https://github.com/electrobrian/green-roomz/releases/tag/v1.0.0 pending merge to tag upstream.
